package.json

```diff
@@ -17,6 +17,7 @@
   "license": "MIT",
   "dependencies": {
     "amphora-fs": "^1.0.1",
+    "amphora-passport": "file:../amphora-passport",
```

Comment:
We'll need to do a release of the module for testing. Whenever that work is done can you release a `v0.0.1` of the package and we'll test it out?